### PR TITLE
A bunch of additions 

### PR DESCRIPTION
**Pull request by StoneMcYT**

### CODE REVIEW

**Review by Kira272921 — CHANGES_REQUESTED** [review]
Use `tenor` links for gifs and `imgur` for images :shipit:

File: data/dog/dog.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+		"image": ""
```
Comment (Kira272921) [inline data/dog/dog.json:3]: :shipit: why is there is empty `value` ?

**Review by Kira272921 — CHANGES_REQUESTED** [review]
:octocat:

**Comment by StoneMcYT** [comment]
Still a WiP

File: data/meme/meme.json [hunk 2]
```diff
@@ -16,5 +16,53 @@
...
+        "image": "https://c.tenor.com/NDtMAS77YQ0AAAAi/meme-triggered.gif"
+    },
+    {
+        "image": "https://c.tenor.com/NDtMAS77YQ0AAAAi/meme-triggered.gif"
+    },
+    {
+        "image": "https://img.ifunny.co/images/cd46ae4f7161b20958dce9975d6cca70a523d290a92f7a14d14ca93ed3159564_1.webm"
+    },
+    {
+        "image": "https://i.imgur.com/Vlk1Gc4.mp4"
```
Comment (Kira272921) [inline data/meme/meme.json:33]: no MP4 pleaze ;-;

File: data/meme/meme.json [hunk 3]
```diff
@@ -16,5 +16,53 @@
...
+        "image": "https://i.kym-cdn.com/entries/icons/original/000/038/199/cover5.png"
+    },
+    {
+        "image": "https://i.imgur.com/0QwfQkD.jpeg"
+    },
+    {
+        "image": "https://imageproxy.ifunny.co/crop:x-20,resize:640x,quality:90x75/images/079491dfd506a06769eb210e3cc4057b85ae40133a610499bffc9db0ad07531d_1.jpg"
+    },
+    {
+        "image": "https://i.imgur.com/1yFXNZw.mp4"
```
Comment (Kira272921) [inline data/meme/meme.json:48]: whyyyyyy ;-;

File: index.js [hunk 4]
```diff
@@ -39,5 +39,9 @@ app.get('/api/v1/meme', (req, res) => {
     res.json(randomObject(dataMeme))
 })
 
+app.get('/api/v1/Dog', (req, res) => {
```
Comment (Kira272921) [inline index.js:42]: make the endpoint lowercase, coz lowercase is kul

**Review by Kira272921 — COMMENTED** [review]
:trollface:

File: index.js [hunk 5]
```diff
@@ -5,7 +5,8 @@ const path = require('path')
 // JSON Data
 const dataAnimePat = require('./data/anime/pat/pat.json')
 const dataAnimeDance = require('./data/anime/dance/dance.json')
-const dataCat = require('./data/cat/gif.json')
+const dataCat = require('./data/cat/cat.json')
+const dataDog = require('./data/dog/dog.json')
```
Comment (Kira272921) [inline index.js:9]: import the fox and bird JSON files as well

**Review by Kira272921 — COMMENTED** [review]
@StoneMcYT import the JSON files into index.js as well when you create new endpoint

**Pull request merged**